Update dashboard-statement-details.md (#12575)
pingcap/docs: dashboard/dashboard-statement-details.md

- Queries that access TiFlash
- Queries that join three or more tables

> To use this feature, you must have the SUPER privilege. If you encounter permission issues while using it, refer to [TiDB Dashboard User Management](dashboard-user.md) to add the necessary privileges.

Reviewer: ```suggestion
To use this feature, you must have the SUPER privilege. If you encounter permission issues while using it, refer to [TiDB Dashboard User Management](/dashboard/dashboard-user.md) to add the necessary privileges.
```

> The current feature requires users to have SUPER privilege to use. If you encounter permission issues while using it, please refer to [TiDB Dashboard User Management](dashboard-user.md) to add the necessary privileges.

Reviewer: ```suggestion
To use this feature, you must have the `SUPER` privilege. If you encounter permission issues while using it, refer to [TiDB Dashboard User Management](dashboard-user.md) to add the necessary privileges.
```
